solution/ui/regulations/js/src/components/SupplementalContent.vue

```diff
@@ -37,8 +37,10 @@ import SupplementalContentCategory from "./SupplementalContentCategory.vue";
 import {
     getSupplementalContentByCategory,
     v3GetSupplementalContent,
+    getSubPartsForPart
 } from "../../api";
 import { EventCodes, formatResourceCategories } from "../../utils";
+import { getSubpartTOC } from '../../../eregs-vite/src/utilities/api';
```

Comment:
This is the wrong API file.  If it isn't in the API file referenced in line 41 above, you need to implement it there.  It is probably a small function and not much to copy over if it isn't there.  This is why the build is breaking though